src/kixi/lapita/plot.clj

```diff
@@ -0,0 +1,101 @@
+(ns kixi.lapita.plot
+  (:require [thi.ng.geom.viz.core :as viz]
+            [thi.ng.geom.svg.core :as svg]
+            [clojure.core.matrix.dataset :as ds]
+            [witan.datasets :as wds]))
+
+;; Implement example from thi.ng/geom
+;; https://github.com/thi-ng/geom/blob/master/geom-viz/src/core.org#bar-graph
+
+(defn export-viz
+  [viz path plot-width plot-height]
+  (->> viz
+       (svg/svg {:width plot-width :height plot-height})
+       (svg/serialize)
+       (spit path)))
+
+(defn transform-data-for-plot
+  [ds x-data y-data]
+  (map vector x-data y-data))
+
+(defn calc-width-of-bar
+  [x-data]
+  (cond
+    (<= (count x-data) 5) 70
+    (and (> (count x-data) 5) (<= (count x-data) 20)) 20
+    :else 10))
+
+(defn bar-spec
+  [data num width]
+  (fn [idx col]
+    {:values     data
+     :attribs    {:stroke       col
+                  :stroke-width (str (dec width) "px")}
+     :layout     viz/svg-bar-plot
+     :interleave num
+     :bar-width  width
+     :offset     idx}))
+
+(defn viz-spec
+  [x-data y-data plot-width plot-height]
+  {:x-axis (viz/linear-axis
+            {:domain [(let [min-x (reduce min x-data)]
+                        (if (= min-x 0) 0 (dec min-x)))
```

Comment:
I think it works well enough for now with negative x-values

![test-plot](https://cloud.githubusercontent.com/assets/2502761/23360553/e10493d6-fce3-11e6-8b2a-128ad21b43a5.png)